rs/orchestrator/src/registration.rs

```diff
@@ -222,6 +222,7 @@ impl NodeRegistration {
             // Unused section follows
             p2p_flow_endpoints: Default::default(),
             prometheus_metrics_endpoint: Default::default(),
+            node_type: None,
```

Comment:
To clarify, is this node_type the [node remuneration type](https://wiki.internetcomputer.org/wiki/Node_Provider_Remuneration#Remuneration_models)? 

If so, I would try to be explicit about this in the naming so as not to confuse with other "node types" (IC-OS variant (host/guest), ic-os dev/prod variants, hardware generation (gen1/gen2/gen3), hardware vendor (dell, supermicro, asus, etc.))

Reply:
yes, good point, I will rename the field.